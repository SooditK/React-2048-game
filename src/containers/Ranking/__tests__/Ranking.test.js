import React from 'react';
import renderer from 'react-test-renderer';
import { Rank } from '../index';

describe('<Ranking />', () => {
  it('component render', () => {
    const ranking = renderer
      .create(<Rank onGetRankingList={() => {}} list={[]} />)
      .toJSON();
    expect(ranking).toMatchSnapshot();
  });
});
