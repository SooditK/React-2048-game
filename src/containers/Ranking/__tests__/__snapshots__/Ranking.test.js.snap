// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Ranking /> component render 1`] = `
<div
  className="main"
>
  <div
    className="echarts"
    id="echarts"
  />
</div>
`;
